lib/pyfrc/sim/ui.py

```diff
@@ -20,7 +20,7 @@
 from .ui_widgets import CheckButtonWrapper, PanelIndicator, Tooltip, ValueWidget
 
 
-#from hal import TalonSRXConst as tsrxc
+from ctre._imple.constants import Constants as tsrxc
```

Comment:
There needs to be a

```
try:
    import ctre
except ImportError:
    ctre = None
```

here, otherwise the sim will fail without robotpy-ctre installed.

Also, it's `ctre._impl`, not `_imple`